src/Microsoft.Android.Build.BaseTasks/AsyncTask.cs

```diff
@@ -157,40 +157,6 @@ public void LogError (string message)
 		public void LogError (string message, params object [] messageArgs)
 			=> LogCodedError (code: null, message: string.Format (message, messageArgs));
 
-		public void LogErrorFromException (Exception exception)
```

Comment:
This API might be needed at some point by classes deriving from AsyncTask. Its part of the public API.

Reply:
This is only defined in the [android-sdk-installer copy of AsyncTask](https://github.com/xamarin/android-sdk-installer/blob/7455826102c50db08e34b044aa5348b8b4d459fe/Xamarin.Installer.Build.Tasks/Tasks/AsyncTask.cs#L167), which I was planning to remove after this addition is merged and bumped. I think we should be able to use the [AsyncTask specific extension](https://github.com/dotnet/android-tools/blob/1ea4e35eaf3fe09b9ca8adf0a29b4d9988146338/src/Microsoft.Android.Build.BaseTasks/UnhandledExceptionLogger.cs#L16) instead, and remove this one specific call site if that seems fair?

Reply:
That might be ok. I just want to make sure we are covered because if people use `Log.XXX` based methods in a Task it will lock up.